mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -45,6 +45,7 @@ CLASS none/ayr net/minecraft/block/material/Material
 	METHOD <init> (Lnone/ays;)V
 		ARG 0 color
 	METHOD a isFullBlock ()Z
+	METHOD b blocksLight ()Z
```

Comment:
This is separate from light emitting (torches, glowstone). This one means that the block material does not let light pass through it (unlike glass), for purposes of light calculations.